modules/itc/src/main/scala/lucuma/itc/ItcChart.scala

```diff
@@ -11,21 +11,36 @@ import io.circe.Encoder
 import io.circe.HCursor
 import io.circe.Json
 import io.circe.generic.semiauto._
+import lucuma.core.syntax.string._
+import lucuma.core.util.Enumerated
 
 import scala.util.Try
 
 enum SeriesDataType:
   case SignalData, BackgroundData, SingleS2NData, FinalS2NData, PixSigData, PixBackData
 
-given Decoder[SeriesDataType] = (c: HCursor) =>
-  Decoder.decodeJsonObject(c).flatMap { str =>
-    val key = str.keys.headOption.orEmpty
-    Try(SeriesDataType.valueOf(key)).toEither.leftMap { _ =>
-      DecodingFailure(s"no enum value matched for $key", List(CursorOp.Field(key)))
-    }
-  }
+object SeriesDataType:
+  // Don't make it a given or it will produce incompatible encoder/decoders with respect to the old itc
+  private val enumerated: Enumerated[SeriesDataType] =
+    Enumerated.of(
+      SeriesDataType.SignalData,
+      SeriesDataType.BackgroundData,
+      SeriesDataType.SingleS2NData,
+      SeriesDataType.FinalS2NData,
+      SeriesDataType.PixSigData,
+      SeriesDataType.PixBackData
+    )
+
+  given Encoder[SeriesDataType] = (a: SeriesDataType) =>
+    Json.fromString(enumerated.tag(a).toScreamingSnakeCase)
 
-given Encoder[SeriesDataType] = Encoder.encodeString.contramap(_.toString)
+  given Decoder[SeriesDataType] = (c: HCursor) =>
+    Decoder.decodeJsonObject(c).flatMap { str =>
+      val key = str.keys.headOption.orEmpty
+      Try(SeriesDataType.valueOf(key)).toEither.leftMap { _ =>
+        DecodingFailure(s"no enum value matched for $key", List(CursorOp.Field(key)))
+      }
+    }
```

Comment:
Since `Enumerated` instances define a circe codec, would this work:

```scala
given Encoder[SeriesDataType] = enumerated

given Decoder[SeriesDataType] = enumerated
```

Reply:
I think so.

Reply:
The encoder is fine but not the decoder. It needs to match what comes from the old itc which uses a slightly different encoding

Reply:
It works for the encoder

Reply:
I see.  Does this mean that it doesn't round-trip?

Reply:
It won't, maybe I should model this as two enums with the same members, one to talk to the old ITC and one to provide the graphql output

Reply:
or just two codecs i guess, at least one not implicit

Reply:
Right that sounds good